holoviews/core/data.py

```diff
@@ -1308,9 +1334,288 @@ def aggregate(cls, columns, kdims, function, **kwargs):
 
 
 
+class GridColumns(DictColumns):
+    """
+    Interface for simple dictionary-based columns format. The dictionary
+    keys correspond to the column (i.e dimension) names and the values
+    are collections representing the values in that column.
+    """
+
+    types = (dict, OrderedDict, cyODict)
+
+    datatype = 'grid'
+
+    @classmethod
+    def reshape(cls, eltype, data, kdims, vdims):
+        if kdims is None:
+            kdims = eltype.kdims
+        if vdims is None:
+            vdims = eltype.vdims
+
+        if not vdims:
+            raise ValueError('GridColumns interface requires at least '
+                             'one value dimension.')
+
+        dimensions = [d.name if isinstance(d, Dimension) else
+                      d for d in kdims + vdims]
+        if isinstance(data, tuple):
+            data = {d: v for d, v in zip(dimensions, data)}
+        elif not isinstance(data, dict):
+            raise ValueError('GridColumns must be instantiated as a '
+                             'dictionary or tuple')
+
+        if 'vdims' in data:
```

Comment:
I think we might want to change this. Although the functionality is useful, I don't like the arbitrary name 'vdims'. Instead I propose that the dictionary key is allowed to be a tuple of the actual value dimension names e.g ``('height', 'pressure', 'temperature').